swingset/pom.xml

```diff
@@ -173,6 +173,12 @@
 	</dependencies>
 
 	<build>
+		<resources>
```

Comment:
So this modification to the pom is telling it to dynamically modify the swingsetVersion.properties during build and replace ${project.version} with the {version} variable value at the top of the pom? Cool.

Reply:
All the files (both of them) in the directory are filtered/copied to the jar. It didn't seem worth it to create another resource directory.